Buildings/Applications/DHC/EnergyTransferStations/Examples/CoolingIndirectClosedBuildingLoop.mo

```diff
@@ -0,0 +1,230 @@
+within Buildings.Applications.DHC.EnergyTransferStations.Examples;
+model CoolingIndirectClosedBuildingLoop
+  "Example model for indirect cooling energy transfer station that has a closed chilled water loop on the building side"
+  extends Modelica.Icons.Example;
+
+  package Medium = Buildings.Media.Water;
+
+  parameter Modelica.SIunits.MassFlowRate mDis_flow_nominal = 0.5
+    "Nominal mass flow rate on district-side (primary)";
+  parameter Modelica.SIunits.MassFlowRate mBui_flow_nominal = 0.5
+    "Nominal mass flow rate on building-side (secondary)";
+
+  parameter Modelica.SIunits.SpecificHeatCapacity cp=
+    Medium.specificHeatCapacityCp(
+    Medium.setState_pTX(Medium.p_default, Medium.T_default, Medium.X_default))
+    "Specific heat capacity of medium 1";
+
+  Modelica.Blocks.Interfaces.RealOutput TApp(
```

Comment:
We may not need to create the connectors in validation and example models. so we can show `TAppCal1.y`, rather than `TApp`. 
So does other connectors in the `example` folder.

Reply:
No problem. These outputs have been removed.